test/com/ableton/JenkinsMocksTest.groovy

```diff
@@ -13,4 +14,12 @@ class JenkinsMocksTest extends BasePipelineTest {
   void setUp() {
     super.setUp()
   }
+
+
+  @Test
+  void isUnix() throws Exception {
+    // It would be pretty redundant to basically re-implement this method in its own test
+    // case, so instead we just assert the call to see that it didn't go haywire.
+    assert JenkinsMocks.isUnix()
```

Comment:
What will happen if I run the tests on Windows?

Reply:
Ahh, good catch. I'll refactor.